### ISSUE TITLE
`gvar.sample` does not have an svdcut by default in v13.0

### ISSUE THREAD

**Issue by Gattocrucco** [gplepage/gvar#43]
I noticed some of my scripts using `gvar.sample` are failing due to nan samples, with `gvar.sample` complaining that the matrix is not pos def.

Minimal example:

```python
import gvar
import numpy as np

print(gvar.__version__)
gvar.sample(gvar.gvar(np.zeros(2), np.ones((2, 2))))
```

```
13.0
/Users/giacomo/Documents/Scuola/lsqfitgp/Repository/lsqfitgp/pyenv/lib/python3.11/site-packages/gvar/__init__.py:955: UserWarning: Matrix not positive definite, lambda_min/lambda_max = 0; need svdcut?
  gflat, i_wgts, i_invwgts = svd(gflat, svdcut=svdcut, noise=noise, wgts=True)
/Users/giacomo/Documents/Scuola/lsqfitgp/Repository/lsqfitgp/pyenv/lib/python3.11/site-packages/gvar/__init__.py:955: RuntimeWarning: divide by zero encountered in log
  gflat, i_wgts, i_invwgts = svd(gflat, svdcut=svdcut, noise=noise, wgts=True)
/Users/giacomo/Documents/Scuola/lsqfitgp/Repository/lsqfitgp/pyenv/lib/python3.11/site-packages/gvar/__init__.py:955: RuntimeWarning: divide by zero encountered in scalar divide
  gflat, i_wgts, i_invwgts = svd(gflat, svdcut=svdcut, noise=noise, wgts=True)
```

The docstring states:

```
    svdcut (float): If nonzero, singularities in the correlation
        matrix are regulated using :func:`gvar.regulate`
        with an SVD cutoff ``svdcut``. Default is ``svdcut=1e-12``.
```

Which indeed was the default behavior before v13.0.

Did you intend to drop that default? I like `gvar.sample` to do something robust by default.

**Comment by gplepage** [comment]
It is a mistake. I will fix it and put up a new version later this morning. Thanks for pointing it out.

**Comment by gplepage** [comment]
It should be fixed in v13.0.1. Thanks again.

**Comment by Gattocrucco** [comment]
It works, thanks.
